tools/test.py

```diff
@@ -740,6 +743,12 @@ def Execute(args, context, timeout=None, env={}, faketty=False):
   for key, value in env.iteritems():
     env_copy[key] = value
 
+  preexec_fn = None
+  if disable_core_files:
+    def disableCoreFiles():
+      resource.setrlimit(resource.RLIMIT_CORE, (0,0))
```

Comment:
I think this is where I got stuck, the second `0` might have side effects, but on the other hand might not be necessary.
(I say "might" because it's not trivial for me to test)

Reply:
This function only gets executed in the subprocess.

I made this determination by using the debugger to confirm that `disabled_core_files` is `True` when appropriate on line 747 while a breakpoint on line 749 never triggers and `resource.getrlimit(resource.RLIMIT_CORE)` (executed from the main process after the test has run) does not return `(0, 0)` (and it does if I run `resource.setrlimit(resource.RLIMIT_CORE, (0,0))` in the parent process elsewhere).

Reply:
AFAIK it is `pickle`d which is not that far from `Function.prototype.toString()`